src/adapters/bidderFactory.js

```diff
@@ -249,7 +249,7 @@ export function newBidder(spec) {
               typeof request.data === 'string' ? request.data : JSON.stringify(request.data),
               {
                 method: 'POST',
-                contentType: 'text/plain',
+                contentType: 'application/json',
```

Comment:
Thanks for the PR. I think we want to make this configurable however. Reason being that if it's `application/json` then CORS preflight request is required (additional round trip). `text/plain` doesn't have this requirement.

Reply:
Thanks, is the intention for 1.0.0 then for the endpoint to accept the POST request JSON payload in plain text format?

Reply:
how do you feel about an optional 'contentType' param on the spec object?

So it would be,
```
contentType: spec.contentType || 'text/plain'
```

Reply:
The `spec` probably isn't a great place, because some adapters make several server requests. If a bidder wanted to make two requests with two different content types, they'd be stuck.

It would fit in pretty nicely on the [ServerRequest](https://github.com/prebid/Prebid.js/blob/master/src/adapters/bidderFactory.js#L64) object, though.

Reply:
works for me, thanks for helping out. We are just trying to avoid our current method of sending JSONP.

Reply:
I'll get a new PR in shortly.

Reply:
yep yep... the internet thanks you :)